lib/rules/attribute-order.js

```diff
@@ -178,6 +193,33 @@ export default class AttributeOrder extends Rule {
       : AttributeType.ATTRIBUTES;
   }
 
+  getAttribute(nodeTokens, { type, name, value }) {
```

Comment:
Can you add a high-level sentence ahead of each function to explain what it does?

Reply:
Refactored for clarity and added docs to most functions.